apache/wsgi.conf.in

```diff
@@ -101,33 +133,33 @@ Header always set Access-Control-Allow-Headers "x-requested-with, Content-Type,
 # assign the WSGI app instance the process group defined aboven, we put the WSGI
 # app instance in the global application group so it is always executed within
 # the main interpreter
-<Directory ${current_directory}/apache>
+<Directory ${INSTALLDIR}/apache>
     Order allow,deny
     Allow from all
-    WSGIProcessGroup mf-chsdi3:${apache_base_path}
+    WSGIProcessGroup mf-chsdi3:${APACHE_BASE_PATH}
     WSGIApplicationGroup %{GLOBAL}
 </Directory>
 
 # Some services are not "free": control is done at varnish level
-<LocationMatch ^${apache_entry_path}/rest/(height|profile)>
+<LocationMatch ^${APACHE_ENTRY_PATH}/rest/(height|profile)>
    Order Deny,Allow
    Allow from all
 </LocationMatch>
 
 # Static for cross domain flash/arcgis
-<LocationMatch ^${apache_entry_path}/files>
+<LocationMatch ^${APACHE_ENTRY_PATH}/files>
    Header unset Access-Control-Allow-Methods
    Header always set Access-Control-Allow-Methods "POST, GET, DELETE, OPTIONS"
 </LocationMatch>
 
 # kml file downloads
-AliasMatch ^${apache_entry_path}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${kml_temp_dir}/$1
+AliasMatch ^${APACHE_ENTRY_PATH}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${KML_TEMP_DIR}/$1
 
 # KML admin
 RewriteCond %{HTTP:X-Forwarded-Proto} !https
-RewriteRule ^${apache_entry_path}/admin         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
+RewriteRule ^${APACHE_ENTRY_PATH}/admin         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
 
-<LocationMatch ^${apache_entry_path}/(admin/kml|static/info.json)>
+<LocationMatch ^${APACHE_ENTRY_PATH}/(admin/kml|static/info.json)>
     AuthType Basic
     AuthName "Your IWI credential"
     # Optional line:
```

Comment:
beware, I didn't see a COPY of 
```
    AuthUserFile "/etc/apache2/htpasswd"
    AuthGroupFile "/etc/apache2/htgroup"
```
so the admin interface for KML will probably not work (unless you mount this from anoter container/folder on host)

Reply:
Hm, the admin KML interface is pretty much dead now.
And credentials should be check elsewhere, the same way we use for height/profile